documentation/staging/content/managing-domains/debugging.md

```diff
@@ -28,7 +28,8 @@ For more information, see [Domain events]({{< relref "/managing-domains/accessin
 If your introspector job failed, then examine the `kubectl describe` of the job and its pod, and also examine its log, if one exists.
 
 {{% notice tip %}}
-To prevent the introspector job from retrying while you are debugging a failure, set the operator's Helm `domainPresenceFailureRetryMaxCount` parameter to `0`. For more information, see the [Configuration reference]({{< relref "/managing-operators/using-helm#domainpresencefailureretrymaxcount-and-domainpresencefailureretryseconds" >}}).
+To prevent the introspector job from retrying while you are debugging a failure, configure `domain.spec.failureRetryLimitMinutes` to `0`.
+For more information, see [Domain failure retry processing]({{< relref "/managing-domains/domain-lifecycle/retry.md" >}}).
```

Comment:
Consider updating [Domain events] to mention the retry doc in place(s) that make sense.

(There's also going to be a new conditions doc that probably should do same.)

Reply:
Right. We should. Thanks. Added references to this doc from a couple of places in the domain events doc.